app/presenters/publishing_api/how_government_works_presenter.rb

```diff
@@ -0,0 +1,94 @@
+module PublishingApi
+  class HowGovernmentWorksPresenter
+    attr_accessor :update_type
+
+    def initialize(update_type: nil)
+      self.update_type = update_type || "major"
+    end
+
+    def content_id
+      "f56cfe74-8e5c-432d-bfcf-fd2521c5919c"
+    end
+
+    def content
+      content = BaseItemPresenter.new(
+        nil,
+        title: "How government works",
+        update_type:,
+      ).base_attributes
+
+      content.merge!(
+        base_path:,
+        description: "About the UK system of government. Understand who runs government, and how government is run.",
+        document_type: "how_government_works",
+        public_updated_at: Time.zone.now,
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "how_government_works",
+        details:,
+      )
+
+      content.merge!(PayloadBuilder::Routes.for(base_path))
+    end
+
+    def base_path
+      "/government/how-government-works"
+    end
+
+    def links
+      return {} if reshuffle_in_progress?
+
+      {
+        current_prime_minister: [Role.find("prime-minister").current_role_appointment.person.content_id],
+      }
+    end
+
+    def details
+      if reshuffle_in_progress?
+        {
+          reshuffle_in_progress: reshuffle_in_progress?,
+        }
+      else
+        {
+          department_counts:,
+          ministerial_role_counts:,
+          reshuffle_in_progress: reshuffle_in_progress?,
+        }
+      end
+    end
+
+    def department_counts
+      {
+        ministerial_departments: Organisation.listable.ministerial_departments.count,
+        non_ministerial_departments: Organisation.listable.non_ministerial_departments.count,
+        agencies_and_public_bodies: Organisation.listable.select { |o| o.type.agency_or_public_body? }.count,
```

Comment:
I note that we were hard-coding this before, so this seems like a good improvement. How are we going to display this on the frontend? Round down to nearest hundred and say x+? (I.e. at the moment we have 444, so does this go to saying "400+ agencies" rather than the current "300+"?

Reply:
My plan was to have logic in the frontend so if it's less than 300, show the actual number.  If it's more, we'll show 300+.  But I thought having the actual number in the content item will make it easier to change this in the future.

Reply:
Yeah I vote for rounding personally, but we can see when we get to that bit 👍